SayTheirNames/Source/Controller/Donations/DonationsMoreDetails/DonationsMoreDetailsController.swift

```diff
@@ -181,5 +203,21 @@ final class DonationsMoreDetailsController: UIViewController {
     @objc func shareAction(_ sender: Any) {
         // TODO: Share button action
     }
-    
+}
+
+// MARK: UICollectionViewDelegate
+
+extension DonationsMoreDetailsController: UICollectionViewDelegate {
+    
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        if indexPath.section == DonationSectionLayoutKind.socialMedia.rawValue {
+            let hashtag = donation.hashTags[indexPath.row]
+            
+            if let url = URL(string: hashtag.link) {
+                let vc = SFSafariViewController(url: url)
+                vc.modalPresentationStyle = .overFullScreen
+                self.present(vc, animated: true)
+            }
```

Comment:
we should open hashtags in the external browser - then they get redirected to the Twitter app